### PR TITLE
Alignment issue fix in About us page

### PR DESCRIPTION
**Pull request by rusiruavb**
Summary
- Get pull from the main branch
- Fix the alignment issue in the hero section
- Fix alignment issue in what we do card items 

**Previous Hero Secion**
Previous

**Changed Hero Section**
Changed

**Previous What we do section**
Card-Previous

**Changed What we do section**
Card-Changed

### CODE REVIEW

**Review by rusiruavb — COMMENTED** [review]
I have reviewed my changes. Please let me know is if there is anything that need to be changed

File: src/modules/AboutUsModules/AboutUsHeroSection/HeroSection.css [hunk 1]
```diff
@@ -248,16 +246,16 @@
     left: 0px;
     top: 0px;
     background: var(--light-background-color);
-    clip-path: polygon(0 0, 100% 0%, 100% 70%, 0 95%);
+    clip-path: polygon(0 0, 100% 0%, 100% 78%, 0 100%);
   }
 
   .overlay {
-    height: 580px;
+    height: 620px;
```
Comment (Shehanka) [inline src/modules/AboutUsModules/AboutUsHeroSection/HeroSection.css:253]: Don't use px, use rem or em

**Review by Shehanka — DISMISSED** [review]

File: src/modules/AboutUsModules/AboutUsHeroSection/HeroSection.css (hunk 1 above)
Comment (rusiruavb) [reply src/modules/AboutUsModules/AboutUsHeroSection/HeroSection.css:244]: Ok,I will change it

Comment (rusiruavb) [reply src/modules/AboutUsModules/AboutUsHeroSection/HeroSection.css]: @Shehanka changed it. Please take a look

**Review by rusiruavb — COMMENTED** [review]
New changes added

**Review by DInuwan97 — APPROVED** [review]

**Review by gihanrcg — APPROVED** [review]

**Pull request merged**